src/pycares/__init__.py

```diff
@@ -65,6 +65,13 @@
     "QUERY_TYPE_SOA" : "T_SOA",
     "QUERY_TYPE_SRV" : "T_SRV",
     "QUERY_TYPE_TXT" : "T_TXT",
+
+    # Query classes
+    "QUERY_CLASS_IN": "C_IN",
+    "QUERY_CLASS_CH": "C_CHAOS",
+    "QUERY_CLASS_HS": "C_HS",
+    "QUERY_CLASS_NONE":"C_NONE",
+    "QUERY_CLASS_ANY": "C_ANY",
 }
 
 for k, v in exported_pycares_symbols_map.items():
```

Comment:
Btw, it could probably be worth to just type these out, if not only to allow auto-completion for IDEs.